daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DarReader.scala

```diff
@@ -21,98 +22,121 @@ class DarReader[A](
     // The `Long` is the dalf size in bytes.
     parseDalf: (Long, InputStream) => Try[A]) {
 
-  import Errors._
+  import DarReader._
 
-  private val manifestEntry = new ZipEntry("META-INF/MANIFEST.MF")
-
-  def readArchive(darFile: ZipFile): Try[Dar[A]] =
+  def readArchive(name: String, darStream: ZipInputStream): Try[Dar[A]] = {
```

Comment:
I'd like to remove the `name` from here as it's only used for texts of errors. I think this module should not be aware of any dar files and if we need to log/return an error containing the dar file name, it should be done in the layer where the file is converted to an `InputStream`. What do you think?

Reply:
We agreed with @gerolf-da to leave this for now as it is.